packages/web-components/src/utils/utils.ts

```diff
@@ -48,3 +48,18 @@ export function consoleDevError(message: string): void {
     console.error(message);
   }
 }
+
+/**
+ * Code needed to trap focus within a modal. Modified from
+ * https://github.com/ionic-team/ionic-framework/blob/main/core/src/utils/overlays.ts#L85
+ * A more extensive list can be found at
+ * https://github.com/KittyGiraudel/focusable-selectors/blob/main/index.js
+ */
+export const focusableQueryString = [
+  '.hydrated:not([tabindex^="-"])',
+  '[tabindex]:not([tabindex^="-"])',
+  'input:not([type=hidden]):not([tabindex^="-"])',
+  'textarea:not([tabindex^="-"])',
+  'button:not([tabindex^="-"])',
+  'select:not([tabindex^="-"])',
+].join(',');
```

Comment:
Could you please move this to a modal.ts or modal-focus.ts file in the utils directory? I'd like to keep this file reserved for functions that may be used by multiple components.

Reply:
Done.